flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -260,6 +263,189 @@
 					text("\"ip\" - uses host's ip address as binding address"))
 				.build());
 
+	// ------------------------------------------------------------------------
+	//  Memory Options
+	// ------------------------------------------------------------------------
+
+	/**
+	 * Total Process Memory size for the TaskExecutors.
+	 */
+	public static final ConfigOption<String> TOTAL_PROCESS_MEMORY =
+		key("taskmanager.memory.total-process.size")
```

Comment:
Maybe
```suggestion
		key("taskmanager.memory.process.size")
```

Reply:
I'm in favor of keep the prefix 'total-', to indicate that this is a coarser level config option that is equivalent to the sum of other options.

Reply:
ok